### PR TITLE
[ADD] developer: add js company service

### PR DESCRIPTION
**Pull request by SimonGenin**

### CODE REVIEW

File: content/developer/reference/frontend/services.rst [hunk 1]
```diff
@@ -131,6 +133,48 @@ Reference List
...
+Company service
+---------------
+
+Overview
+~~~~~~~~
+
+* Technical name: `company`
+* Dependencies: `user`, `router`, `cookie`
+
+Odoo has a concept of companies. You can manage several with the same Odoo instance.
```
Comment (ged-odoo) [inline content/developer/reference/frontend/services.rst:147]: i don't think you should use the word `instance`. it feels like a odoo word that is wrong.

Maybe rewrite the first lines like this:

```
Odoo allows users to manage more than one companies from the same database.
```

File: content/developer/reference/frontend/services.rst [hunk 2]
```diff
@@ -131,6 +133,48 @@ Reference List
...
+but the products of all the active companies are shown. 
+
+This service provides information about them as well as a couple of functions.
+
+API
+~~~
+
+.. js:attribute:: availableCompanies: number[]
+
+  Get all the company ids registered in the odoo instance.
```
Comment (brboi) [inline content/developer/reference/frontend/services.rst:160]: and available to the user ?

File: content/developer/reference/frontend/services.rst [hunk 3]
```diff
@@ -131,6 +133,48 @@ Reference List
...
+---------------
+
+Overview
+~~~~~~~~
+
+* Technical name: `company`
+* Dependencies: `user`, `router`, `cookie`
+
+Odoo has a concept of companies. You can manage several with the same Odoo instance. 
+More than one companies can be *active* at any time, but one company is always considered selected.
```
Comment (aab-odoo) [inline content/developer/reference/frontend/services.rst:148]: company

File: content/developer/reference/frontend/services.rst [hunk 4]
```diff
@@ -131,6 +133,48 @@ Reference List
...
+The selected company is basically considered the current company, but for pages that support it,
+the content of all the active companies are displayed. By example, in the product page, the company A may be the main one
+but the products of all the active companies are shown.
```
Comment (aab-odoo) [inline content/developer/reference/frontend/services.rst:151]: in the backend, I don't think there's a notioni of "pages that support multi company". It's the way all models work, isn't it?

File: content/developer/reference/frontend/services.rst [hunk 5]
```diff
@@ -131,6 +133,48 @@ Reference List
...
+.. js:attribute:: currentCompany: number
+
+  Get the current main company id
+
+.. js:function:: setCompanies(mode, ...companyIds)
+
+  :param string mode: `loginto` to set as main company (the first one of companyIds parameter is taken), `toggle` to toggle the active state of all the companyIds.
+  :param number[] companyIds: the company ids impacted.
+
+Changes the state of the main and active companies.
```
Comment (ged-odoo) [inline content/developer/reference/frontend/services.rst:175]: wrong indentation i think

**Pull request closed without merging**